http/src/main/scala/org/http4s/blaze/http/http2/HeaderDecoder.scala

```diff
@@ -0,0 +1,91 @@
+package org.http4s.blaze.http.http2
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets.US_ASCII
+
+import org.http4s.blaze.http.http2.Http2Exception._
+import org.http4s.blaze.http.http2.Http2Settings.DefaultSettings
+import org.http4s.blaze.util.BufferTools
+import com.twitter.hpack.{Decoder, HeaderListener}
```

Comment:
Is this a new dependency?  I didn't notice it sneaking in.  I'm not too fussed about it since it's slated for removal.

Reply:
No, it had always been there. Unfortunately it's no longer maintained as it was folded into the Netty project. That said, it still works as expected so there is little stunning force to migrate to a new one.